### PR TITLE
Docker setup for local development

### PR DESCRIPTION
**Pull request by Bashamega**
## Fixes Issue
Some people prefer running Postgres in docker, rather than installing it locally.
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
Can you add the label `hacktoberfest-accepted`?

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+version: "3"
+services:
+  db:
+    image: postgres:latest # You can specify a version, e.g., postgres:13
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
```
Comment (eddiejaoude) [inline docker-compose.yml:10]: I think the `.env.example` db string will need to be updated to match?

Comment (Bashamega) [reply docker-compose.yml:10]: Correct. I will fix it

Comment (Bashamega) [reply docker-compose.yml:10]: done

Comment (eddiejaoude) [reply docker-compose.yml:10]: thank you

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**